Xamarin.Forms.DataGrid/DataGrid.xaml.cs

```diff
@@ -101,8 +101,14 @@ void HandleItemsSourceCollectionChanged(object sender, NotifyCollectionChangedEv
 			if (SelectedItem != null && !InternalItems.Contains(SelectedItem))
 				SelectedItem = null;
 		}
-
-		public static readonly BindableProperty RowHeightProperty =
+        public static readonly BindableProperty HasUnevenRowsProperty =
```

Comment:
Entire project uses tabs for indentation. Please follow the rule and convert spaces to tabs.